packages/xhost.rb

```diff
@@ -0,0 +1,34 @@
+require 'package'
+
+class Xhost < Package
+  description 'Server access control program for X'
+  homepage 'https://www.x.org/archive/X11R6.8.1/doc/xhost.1.html'
```

Comment:
See my comment on the previous PR. There is no unversioned page.

Reply:
> `X11R6.8.1` is just the revision of the X server. No correlation to the version of the package or its contents. In fact, I believe that this page is the the only revision of the X server that contains that manpage. It was split out into a separate package/executable in later revisions.

Reply:
It doesn't matter anyway, no one goes to a homepage for a package.

Reply:
I didn't see any problem with the homepage on the previous PR.  The Github project looks up to date also.  Sure it doesn't have a pre-made configure script but does that really matter?

Reply:
Yes. It does. Why would anybody look at the code for a package if they want an explanation? A manpage would be just perfect for that.

Reply:
Is "Latest commit 28015d9  on Jan 25, 2017" up to date?

Reply:
The GitHub project has been abanoned. The source code is on GitLab now, however why use that when there is a pre-packaged source tarball with a `configure` script?

Reply:
Your version was released on.... 

2015-04-17
--

Reply:
I don't even know what you are arguing about.  That commit is newer that the version you are suggesting so I'm not following.

Reply:
All right, but I want a **stable** release. Is that wrong?

Reply:
I know because I picked the stable release. We are arguing semantics right now. There's no difference in either version that we picked. You were just arbitrarily wanting to change things that worked fine before.

Reply:
It's the same release and the README.md has even more good information about the project than your _"manpage"_ homepage.

Reply:
You must be an only child.

Reply:
How do you know this Github project is abandoned?  I don't see anything saying it has been.

Reply:
Do you know how insulting that is to me? I don't think this relates to the issue at hand.

Reply:
Maybe it's insulting because it's true?  I told you not to self resolve conversations.

Reply:
I resolved it because we were getting off-topic and the discussion was getting out of hand. Please don't make comments about my home life.

Reply:
What is off topic?  I just don't want a manpage for a homepage.  That doesn't make sense and especially if it is versioned.

Reply:
The manpage is the only one available from x.org, and it isn't strictly "versioned". That's just the last revision of the X server it was released with. If you can find another homepage from x.org that isn't versioned and provides useful information, I'm all for it.

Reply:
I can see you have no intention of changing this so I'll just merge and change when I generate the pre-built binaries.

Reply:
I have intention of changing this if you so wish, but I need a valid page to change it to.